languages/java/exercises/concept/classes/.meta/src/reference/java/RemoteControlCar.java

```diff
@@ -0,0 +1,34 @@
+class RemoteControlCar
+{
```

Comment:
normally in Java left curly bracket is on the same line of the class name
so 
```
class Car {
}
```
This is valid for all the code in this page
This is valid for block of code too